package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dreamjs",
-  "version": "0.1.7",
+  "version": "0.1.9",
```

Comment:
Why was version number changed, and jumped over 0.1.8?
Usually the package author does that.